app/src/main/java/org/mozilla/focus/utils/UrlUtils.kt

```diff
@@ -151,4 +151,9 @@ object UrlUtils {
 
         return host.substring(start)
     }
+
+    @JvmStatic
+    fun toDisplayUrl(url: String): String {
+        return ""
```

Comment:
nit: when adding code that is going to change, I usually add a TODO to notify my reviewer that I'm going to update it.

Reply:
Good idea.  I'll do that in the future.